<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarkUpWidget</class>
 <widget class="QWidget" name="MarkUpWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mark-Up</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
       <item>
        <widget class="MarkUpView" name="_entryView"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="CGTVideoControls" name="_entryControls" native="true"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
       <item>
        <widget class="MarkUpView" name="_cloneView"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="CGTVideoControls" name="_cloneControls" native="true"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLCDNumber" name="_entryFrameNumber">
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="_regionsBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="_entryComboBox">
       <item>
        <property name="text">
         <string>Line</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Point</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_helpButton">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Jump to Key-Frame</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QComboBox" name="_keyFrameBox"/>
       </item>
       <item>
        <widget class="QRadioButton" name="_cloneButton">
         <property name="text">
          <string>Clone</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">_cloneButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="_deleteCloneButton">
         <property name="text">
          <string>Delete</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">_cloneButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="_cloneFrameNumber">
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MarkUpView</class>
   <extends>QGraphicsView</extends>
   <header>cgt.gui.markupview</header>
  </customwidget>
  <customwidget>
   <class>CGTVideoControls</class>
   <extends>QWidget</extends>
   <header>cgt.gui.cgtvideocontrols</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>_entryComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MarkUpWidget</receiver>
   <slot>entry_view_set_marker_type(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_cloneButtonGroup</sender>
   <signal>buttonClicked(QAbstractButton*)</signal>
   <receiver>MarkUpWidget</receiver>
   <slot>clone_view_set_state(QAbstractButton)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_regionsBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MarkUpWidget</receiver>
   <slot>region_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>605</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_keyFrameBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MarkUpWidget</receiver>
   <slot>jump_to_key_frame(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>769</x>
     <y>609</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_helpButton</sender>
   <signal>clicked()</signal>
   <receiver>MarkUpWidget</receiver>
   <slot>display_help()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>605</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="_cloneButtonGroup"/>
 </buttongroups>
</ui>
